lib/internal/modules/esm/loader.js

```diff
@@ -482,10 +603,7 @@ class ESMLoader {
   }
 
   preload() {
-    const count = this.#globalPreloaders.length;
-    if (!count) return;
-
-    for (let i = 0; i < count; i++) {
+    for (let i = this.#globalPreloaders.length - 1; i > -1; i--) {
```

Comment:
```suggestion
    for (let i = this.#globalPreloaders.length - 1; i >= 0; i--) {
```

Reply:
O.o sure, but why?

Reply:
This is a nit, so feel free to ignore if you feel otherwise. When I first read it I was wondering at the significance of `-1`—was an `indexOf` involved or something?—until I realized you were just counting down to zero. Personally I find it easier to follow “until `i` reaches 0, inclusive” rather than “until `i` reaches -1, exclusive” but it’s subjective.